libgraphics/include/pivot/graphics/types/Vertex.hxx

```diff
@@ -22,6 +22,10 @@ struct Vertex {
     /// @param other The other object to compare
     /// @return true if both object are equal
     bool operator==(const Vertex &other) const noexcept;
+    /// Inequality operator overload
+    /// @param other The other object to compare
+    /// @return true if both object are different
+    bool operator!=(const Vertex &other) const noexcept;
```

Comment:
I think the inequality operator is already available when you overload the equality operator. Can you check whether removing it prevents compilation or causes any warnings ?
```suggestion
```

Reply:
Yep I didn't knew that. It's new from c++20.